### DISCUSSION TITLE
Error d:d:c

### DISCUSSION THREAD

**Discussion by IsmaelARassool** [symfony/symfony#50746] [Q&A]
Hello I'm on windows, with xampp, i'm trying to create a database but I have this problem
![image](https://github.com/symfony/symfony/assets/133125183/ce342406-1d89-4bb0-9bf7-64ac23fb9423)
I already uncommented the extensions pdo and mysql
here is my .env.local
![image](https://github.com/symfony/symfony/assets/133125183/ec614988-a3ac-4f96-b0b2-0d4363c73853)
I also tryed to uninstall everything and reinstall it, tryed to create the database myself and make a d:s:u but same error
Thanks for reading. Have a nice day !

**Comment by ghost** [comment]
Please report this to doctrine/doctrine-bundle

**Reply by IsmaelARassool** [reply]
thank you ! I'm doing it
